ChangeLog

```diff
@@ -1,4 +1,5 @@
 2020-09-03, Version 5.12.0
+	* collectd.conf manpage: Correct description of FQDNLookup option.
```

Comment:
The changelog file will be updated later for the release. You don't have to update it here. The next version would be 5.13.